README.md

```diff
@@ -192,6 +192,8 @@ Current checks:
 
 * Files referenced in config file exist (does not check for secrets files)
 * Deployments contain a `revisionHistoryLimit`
+* CronJobs contain a `JobsHistoryLimit`
+  * History Limits are supported on kubernetes versions greater than 1.6
```

Comment:
Nice! Can you add as well that this means lint would always wind up failing for <1.6 clusters because you can't include `JobHistoryLimit`? The current statement is correct but the implication to users may not be obvious.